server.js

```diff
@@ -11,14 +11,97 @@ mongoose.Promise = Promise;
 // PORT=9000 npm start
 const port = process.env.PORT || 8080;
 const app = express();
+const listEndpoints = require('express-list-endpoints');
 
 // Add middlewares to enable cors and json body parsing
 app.use(cors());
 app.use(express.json());
 
 // Start defining your routes here
 app.get("/", (req, res) => {
-  res.send("Hello Technigo!");
+  const welcomeMessage = "Happy Thoughts API!";
+  const endpoints = listEndpoints(app);
+
+  res.status(200).json({
+    success: true,
+    message: "OK",
+    body: {
+      welcomeMessage,
+      endpoints
+    }
+  });
+});
+
+
+const { Schema } = mongoose;
+const thoughtsSchema = new Schema({
+  message: {
+    type: String,
+    required: true,
+    minlength: 5,
+    maxlength: 140,
+    trim: true
+  },
+  hearts: {
+    type: Number,
+    default: 0,
+  },
+  createdAt: {
+    type: Date,
+    default: Date.now
```

Comment:
I just wrote a thought and looked into your backend, it says the thought is `createdAt": "2023-05-28T16:23:08.969Z"` (2hours "wrong"). I think I have written my code like yours with the `Date.now`, maybe some UTC-syntax can fix that.